### ISSUE TITLE
Same label for positive and negative cases

### ISSUE THREAD

**Issue by nthhiep** [QinbinLi/MOON#8]
Hi,

Thank you for your interesting work. I have a question about the label used for the contrastive loss. Why positive and negative cases have the same label as 0 (lines 309, 297, 303)? It must be 0 for positive and 1 for negative pairs? 

Another question about the result of Scaffold published in your paper. This code does not have Scaffold. So, did you run it from https://github.com/Xtra-Computing/NIID-Bench?

Thank you in advance for your feedback.

**Comment by QinbinLi** [comment]
Hi @nthhiep ,

Note that `logits` is a matrix with 2 columns. The first column is the positive pair and the second column is the negative pair. Then, by setting the label to 0 and using [cross-entropy loss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html), we can compute the contrastive loss.

Yes, you may run Scaffold from https://github.com/Xtra-Computing/NIID-Bench, which is implemented by our understanding on the Scaffold paper.
